nomad/structs/structs.go

```diff
@@ -5610,7 +5614,10 @@ func (p *PeriodicConfig) Validate() error {
 	}
 
 	var mErr multierror.Error
-	if p.Spec == "" {
+	if p.Spec != "" && len(p.Specs) != 0 {
+		_ = multierror.Append(&mErr, fmt.Errorf("You can use only Spec or Specs"))
```

Comment:
I hope that you keep only `Specs` :sweat_smile: 
Because now we use systemd timers, and it's more powerful.
Moreover, even k8s doesn't have multiple cron times.

Reply:
But I understand that it break backward compatibility.